### DISCUSSION TITLE
Variante 11 bzw. 22 - Firmwareversion? Hardwareversion?

### DISCUSSION THREAD

**Discussion by GallusMax** [goecharger/go-eCharger-API-v2#169] [General]
Der (neue?) API key var unterscheidet zwischen der 11kW und 22kW Variante. 

Handelt es sich um unterschiedliche Firmwareversionen, oder unterscheiden sich die Platinen der V3?

Auch Wallboxen, die mit var=11 angezeigt werden, haben den Umschalter, mit dem mit dem go-e CEE32 auf Schuko Adapter etwa den Strom auf 16A begrenzt wird.

**Comment by 0xFEEDC0DE64** [comment]
var gabs bei den V2 boxen damals noch nicht,

wurde mit V3 eingeführt, und zusätzlich wurde die adapter erkennung intern in der box ab fabrik so kurz geschlossen, dass dauerhaft ein adapter erkannt wurde.

bei V4 wird nur mehr mit var limitiert, ohne dass eine adapter erkennung notwendig ist.
die 22kW variante hat entweder den digitalen eingang (fix) oder eine adapter erkennung in der ce dose (fix).
die 11kW mobil variante wertet den digitalen input / adapter erkennung nicht aus und limitiert eben immer auf die 16A (oder was halt im ländersetup in der app eingestellt wird, hängt auch von 1/3 phasig ab)
die 11kW fix oder beide 22kW verfügen über den digitalen Eingang / adapter erkennung und werten diesen aus, falls eingeschaltet (zwangseingeschaltet bei 22kW mobil, damit die adapter erkennung nicht ausgeschaltet werden kann)
